src/ngFormBuilder-complete.js

```diff
@@ -1,5 +1,6 @@
 require('ng-formio/src/formio-complete.js');
 require('angular-drag-and-drop-lists');
+require('../bower_components/ckeditor/ckeditor');
```

Comment:
We should not be including the full ckeditor as a dependency. We just need to ensure that the angular-ckeditor is included in ```ngFormio``` and not just this library.